### DISCUSSION TITLE
looking for a svelte-webpack-tailwind template

### DISCUSSION THREAD

**Discussion by Mic1** [tailwindlabs/tailwindcss#2539] [Help] [answered]
Have to use svelte-webpack rather than svelte-rollup as the mongodb-stitch-sdk plugin does not work with rollup. Lots of googling and found many svelte-rollup-tailwind templates, but not really anything concrete for svelte-webpack-tailwind. Appreciated.

**Comment by swyxio** [comment, marked as answer]
add the preprocessor, then add Tailwind. https://dev.to/swyx/how-to-set-up-svelte-with-tailwind-css-4fg5

**Reply by Mic1** [reply]
Thank you -this works

**Reply by t829702** [reply]
> add the preprocessor, then add Tailwind. https://dev.to/swyx/how-to-set-up-svelte-with-tailwind-css-4fg5

Thanks @sw-yx  for this link, I've searched multiple days on this Tailwindcss together with Svelte, many links were outdated and not always working, I'd definitely like to try your way; am just seeing your link `on Sep 8, 2020 Originally published at swyx.io ・Updated on Jan 7` that looks very fresh

While, would you mind try to push a markdown file to the docs site? then make it official at some link like https://tailwindcss.com/docs/guides/...svelte...sapper...  then it will be maintained and kept up-to-date

- https://tailwindcss.com/docs/guides/vue-3-vite
- https://github.com/tailwindlabs/tailwindcss.com/tree/master/src/pages/docs/guides
  -   angular.mdx create-react-app.mdx gatsby.mdx laravel.mdx nextjs.mdx nuxtjs.mdx rails.mdx vue-3-vite.mdx
